src/views/SignMessage.vue

```diff
@@ -71,12 +63,14 @@ export default class SignMessage extends Vue {
 
         if (this.request.signer) {
             const wallet = this.findWalletByAddress(this.request.signer.toUserFriendlyAddress(), false);
-            if (wallet) {
+            if (wallet && wallet.type !== WalletType.LEDGER) {
                 this.setAccount(wallet.id, this.request.signer.toUserFriendlyAddress(), true);
                 return;
             }
         }
 
+        // if account not specified / found or is an unsupported ledger account let the user pick another account.
+        // Don't automatically reject to not directly leak the information whether an account exists / is a ledger.
```

Comment:
Please use proper capitalization in long-form comments (`If` at the start of the sentence, `Ledger` is a brand name). Also, I plan to add a `forceSigner` option to the signMessage request (#218), so the second line of this comment is/will be untrue in the future.

Reply:
If `forceSigner` is set for a Ledger account, sign message should then redirect to the planned `Ledger not supported for this operation` page.